CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The changelog for `IGListKit`. Also see the [releases](https://github.com/instagram/IGListKit/releases) on GitHub.
 
+### Enhancements
```

Comment:
Just for the sake of completion we should add a header for 2.2.0 with a link to the [milestone](https://github.com/Instagram/IGListKit/milestone/4)

Also can you add a link to this [pull request](https://github.com/Instagram/IGListKit/pull/395) at the end of your contribution, following the same format as other changelog entries